lib/theming/src/themes/light.ts

```diff
@@ -1,36 +1,10 @@
 import { create } from '../create';
-import { color, typography, background } from '../base';
+
+import { themeVars } from './dark-vars';
```

Comment:
@ndelangen should this be from `light-vars`?

Reply:
indeed